### PR TITLE
Export Stackdriver Logging metrics via Prometheus integration

### PR DESCRIPTION
**Pull request by jkohen**

### CODE REVIEW

File: templates/etc/td-agent/td-agent.conf.tmpl [hunk 1]
```diff
@@ -6,6 +6,15 @@
 # https://github.com/GoogleCloudPlatform/fluentd-catch-all-config
 @include config.d/*.conf
 
+# Prometheus monitoring
```
Comment (qingling128) [inline templates/etc/td-agent/td-agent.conf.tmpl]: Add a trailing period?

File: templates/etc/td-agent/td-agent.conf [hunk 2]
```diff
@@ -6,6 +6,15 @@
 # https://github.com/GoogleCloudPlatform/fluentd-catch-all-config
 @include config.d/*.conf
 
+# Prometheus monitoring
```
Comment (qingling128) [inline templates/etc/td-agent/td-agent.conf]: Add a trailing period?

**Comment by jkohen** [comment]
Added periods.

**Comment by qingling128** [comment]
Just FYI, I noticed that this PR was not squashed (We normally choose `Squash and merge` to keep the log history clean.) when being merged. I just squashed the three commits manually.

**Comment by jkohen** [comment]
OK, thanks for letting me know. I got no such guidance before.

**Comment by qingling128** [comment]
No worries. :)

**Pull request merged**